spokestack/asr/google/speech_recognizer.py

```diff
@@ -0,0 +1,104 @@
+"""
+This module contains the google asr speech recognizer
+"""
+import logging
+from queue import Queue
+from threading import Thread
+from typing import Any
+
+import numpy as np  # type: ignore
+from google.cloud import speech  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+_LOG = logging.getLogger(__name__)
+
+
+class GoogleSpeechRecognizer:
+    """ Transforms speech into text using Google's ASR.
+
+     Args:
+         cred_path (str): path to google credentials
```

Comment:
we will also want to support credentials as strings (environment variables, etc.), so that we don't have to put them into files
you can probably deal with this with separate kwargs